pages/apim/3.x/event-native-apim/event-native-apim-introduction.adoc

```diff
@@ -0,0 +1,129 @@
+[[event-native-apim-introduction]]
+= Introduction
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/event_native_apim_introduction.html
+:page-folder: apim/event-native-apim
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+== Event-driven asynchronous APIs
+
+Most developers are familiar with synchronous APIs, where the client sends a request to a server, and the server sends a response back. Every time the client needs new data, it has to send a request. You can think of synchronous APIs as a 'client pull' model. RESTful APIs are a type of synchronous API.
+
+In contrast, asynchronous APIs are event-driven and allow you to stream data. Asynchronous APIs can use callbacks, such as webhooks, if the requested data is not available at the time of the request. You can think of asynchronous APIs as a 'server push' model. AsyncAPIs are a type of asynchronous API.
+
+For more information about event-driven architectures and asynchronous APIs, see the link:https://www.asyncapi.com/[AsyncAPI website].
```

Comment:
AsyncApi website is about AsyncAPI specification,don't think we should rely on it to explain what is an event driven architecture and what is the benefit of it.

Reply:
OK, I will remove this line for now.

As this is a beta release, I don't think the documentation should describe the advantages of an event-driven architecture. Any users of this release should already know. 

The documentation for the full release should cover this.